docs/features/explorer.md

```diff
@@ -139,6 +139,67 @@ Component.Explorer({
 })
 ```
 
+### Use `sort` with pre-defined sort order
+
+Here's another example where a map containing file/folder names is used to define the sort order of the explorer in quartz. All files/folders that aren't listed inside of `nameOrderMap` will appear at the top of that folders hierarchy level. It's also worth mentioning, that the smaller the number set in `nameOrderMap`, the higher up the entry will be in the explorer. Incrementing every folder/file by 100, makes ordering files in their folders a lot easier. Lastly, having the `mapFn` function before or after `sortFn` doesn't matter.
```

Comment:
> Lastly, having the `mapFn` function before or after `sortFn` doesn't matter.

Might be a bit confusing for people, should probably expand to explain that using `mapFn` before or after only doesn't matter so long as only the frontmatter slugs are being used in `sort` or just not mentioned?